packages/server/modules/workspaces/events/eventListener.ts

```diff
@@ -532,10 +571,16 @@ export const workspaceTrackingFactory =
         planName: plan?.name || '',
         planStatus: plan?.status || '',
         planCreatedAt: plan?.createdAt,
-        subscriptionBillingInterval: subscription?.billingInterval,
-        subscriptionCurrentBillingCycleEnd: subscription?.currentBillingCycleEnd,
+        subscriptionCreatedAt,
+        subscriptionBillingInterval,
+        subscriptionCurrentBillingCycleEnd,
         seats,
         seatsGuest: 0,
+        seatsViewerCount,
+        seatsEditorCount,
+        createdAt: workspace.createdAt,
+        projectCount: projectCount[0],
```

Comment:
also, that's now how this works. if you look at the type `GetWorkspacesProjectsCounts` or the implementation of it, you'll see that the return value is a map of `WorkspaceId -> count`, so there's no numbered indexing in there and `projectCount[0]` will always return `undefined`

do make sure to test out that all of this logic actually works as expected, either by actually sending to mixpanel or debugging through